web/src/components/organisms/Dashboard/index.tsx

```diff
@@ -48,11 +50,21 @@ const Dashboard: React.FC<Props> = ({ children, defaultSelectedKeys }) => {
             collapsible
             collapsed={collapsed}
             onCollapse={value => setCollapsed(value)}>
-            <WorkspaceMenu
-              defaultSelectedKeys={defaultSelectedKeys}
-              isPersonalWorkspace={personalWorkspace?.id === currentWorkspace?.id}
-              inlineCollapsed={collapsed}
-              workspaceId={currentWorkspace?.id}></WorkspaceMenu>
+            {projectId && menuType ? (
```

Comment:
Can you think of a better way to conditionally show these two menu types? 

Right now you aren't using the `menuType` in a meaningful way. Even if its `"workspace"`, if there is a `projectId` it'll show the `ProjectMenu~ component, which defeats the purpose of having menuType be `"project" | "workspace"`.

Reply:
@KaWaite , yup the projectId should wrap the two menus, will fix

Reply:
One option is pass the menu component in as a prop.

Reply:
@KaWaite , I thought of passing it, but the whole project has 2 menus, and giving it as a prop will add an unnecessary import and prop passing for each menu usage.

https://github.com/reearth/reearth-cms/blob/6e45c9fb406680bf370be1b073859163593f42ba/web/src/components/organisms/Dashboard/index.tsx#L49-L73